src/components/UI/navbar/index.js

```diff
@@ -100,7 +100,7 @@ const Navbar = () => {
             <div className={styles.navBarLogin}>
               {!isLoggedIn && (
                 <Link href={authUrl}>
-                  <a className={styles.btnLogin}>
+                  <div className={styles.btnLogin}>
```

Comment:
if it is a button why keep it in a div cant we directly keep it inside the button tag itself?

Reply:
I had tried this way, but actually div tag has its own styles inside which there is a button. To keep the styles same, I had to keep the div tag also

Reply:
For clickable elements its preferred to either use button or anchor other tags are not preferred.